api/src/main/scala/dcos/metronome/api/v1/models/package.scala

```diff
@@ -248,4 +249,39 @@ package object models {
   }
 
   implicit lazy val MetronomeInfoWrites: Writes[MetronomeInfo] = Json.writes[MetronomeInfo]
+
+  implicit lazy val MarathonRunSpecWrites: Writes[RunSpec] = new Writes[RunSpec] {
```

Comment:
I don't feel good about exposing RunSpec (Marathon domain object) through Metronome API...

I would rather expose our own domain object that we have under control and that will not change in between marathon version... 

What do you think?

Reply:
I don't feel like we are exposing it...  this is a json transformation.   I did call it runspec on L271.  we definitely shouldn't do that.